cmd/cluster-version-operator-tests/README.md

```diff
@@ -21,4 +21,40 @@ After [installing-ginkgo](https://onsi.github.io/ginkgo/#installing-ginkgo):
 $ ginkgo ./test/...
 ```
 
-The output looks nicer this way.
\ No newline at end of file
+The output looks nicer this way.
+
+
+## Test Names
+
+### Test labels
+
+See [docs](https://github.com/openshift/origin/tree/main/test/extended#test-labels) for details.
+
+### Ownership
+
+* A `[Jira:"Component"]` tag in the test name (e.g., [Jira:"Cluster Version Operator"]) is preferred to claim the ownership of the test. The component comes from [the list of components of Jira Project OCPBUGS](https://issues.redhat.com/projects/OCPBUGS?selectedItem=com.atlassian.jira.jira-projects-plugin:components-page). See [here](https://github.com/openshift/origin/blob/6b584254d53cdd1b5cd6471b69cb7c22f3e28ecd/test/extended/apiserver/rollout.go#L36) for example.
+
+* Or an entry in the [ci-test-mapping](https://github.com/openshift-eng/ci-test-mapping) repository, which maps tests to owning components. See [here](https://github.com/openshift-eng/ci-test-mapping/tree/main/pkg/components/clusterversionoperator) for example.
+
+### Renaming
+
+If a test is renamed, claim it with [specs.Walk](https://pkg.go.dev/github.com/openshift-eng/openshift-tests-extension@v0.0.0-20251024155203-c498f9efe478/pkg/extension/extensiontests#ExtensionTestSpecs.Walk):
+
+```go
+specs = specs.Walk(func(spec *et.ExtensionTestSpec) {
+    if orig, ok := renamedTests[spec.Name]; ok {
+		spec.OriginalName = orig
+	}
+})
+```
+
+### Deleting
+
+If a test is renamed, claim it with [ext.IgnoreObsoleteTests](https://pkg.go.dev/github.com/openshift-eng/openshift-tests-extension@v0.0.0-20251024155203-c498f9efe478/pkg/extension#Extension.IgnoreObsoleteTests)
```

Comment:
Clicking the link results in:

<img width="2262" height="846" alt="Image" src="https://github.com/user-attachments/assets/cc6c130d-60fa-40a0-b6d5-976196e24bcc" />